index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO MICROVERSE</title>
+    <script src="https://kit.fontawesome.com/de9dd03ce2.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <h2 class="my-logo">Aneal Laryea</h2>
+            <i class="fa-solid fa-bars"></i>
+        </header>
+    
+        <section class="headline">
+            <div class="primary-text">
+                <h3>Hey There.
+                    I’m Aneal</h3>
+            </div>
+            <div class="supporting-text">
+                <p>I’am a Software Developer</p>
+            </div>
+            <div class="description">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="social-media">
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-linkedin"></i>
+                <i class="fa-brands fa-angellist"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-solid fa-m"></i>
+            </div>
```

Comment:
- Please for the icons, make use of an unordered list and style appropriately in flex row, ie 
```html
<ul>
 <li> <i class="fa-brands fa-twitter"></i> </li>
<li> .... </li>
...
</ul>
```
- Tip: You may download those icons of each social media individually  directly from the Figma design, and use them in your project

Reply:
@Ginohmk working on the corrections